tests/Unit/Elliptic/Systems/Elasticity/Equations.py

```diff
@@ -4,41 +4,37 @@
 import numpy as np
 
 
-def primal_fluxes(strain, coordinates, constitutive_relation):
-    return constitutive_relation(strain)
+def constitutive_relation_2d(strain, bulk_modulus, shear_modulus):
+    lame_constant = bulk_modulus - 2. / 3. * shear_modulus
+    return -2. * shear_modulus * lame_constant / (
+        lame_constant + 2. * shear_modulus
+    ) * np.trace(strain) * np.eye(2) - 2. * shear_modulus * strain
```

Comment:
Please auto-format the Python file (or have you already?) If you have the `yapf` Python package installed, VSCode should be able to do it.

Reply:
Yes, this is after using `yapf`.